sdlf-cicd/lambda/domain-cicd/src/lambda_function.py

```diff
@@ -164,10 +164,10 @@ def delete_domain_team_role_stack(cloudformation, team):
 
 
 def create_team_repository_cicd_stack(domain, team_name, template_body_url, cloudformation_role):
-    gitlab_url = ssm.get_parameter(Name="/SDLF/GitLab/Url")["Parameter"]["Value"]
-    gitlab_accesstoken = ssm.get_parameter(Name="/SDLF/GitLab/AccessToken")["Parameter"]["Value"]
+    gitlab_url = ssm.get_parameter(Name="/SDLF/GitLab/Url", WithDecryption=True)["Parameter"]["Value"]
+    gitlab_accesstoken = ssm.get_parameter(Name="/SDLF/GitLab/AccessToken", WithDecryption=True)["Parameter"]["Value"]
     repository = f"sdlf-main-{domain}-{team_name}"
-    namespace_id = ssm.get_parameter(Name="/SDLF/GitLab/NameSpaceId")["Parameter"]["Value"]
+    namespace_id = ssm.get_parameter(Name="/SDLF/GitLab/NameSpaceId", WithDecryption=True)["Parameter"]["Value"]
```

Comment:
Minor and I would understand if you decide not to change it, but I would use `/SDLF/GitLab/NamespaceId` (lower-case `s` as it is written on GitLab)

Reply:
fixed to NamespaceId